admin_manual/installation/examples/lets_encrypt/delete.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+LE_PATH="/opt/letsencrypt"
+LE_CB="certbot"
```

Comment:
ATTENTION:
in /opt/letsencrypt it is ``certbot-auto``

Reply:
Yeah, you're right.